### PR TITLE
Update set-up-atp-safe-links-policies.md

### PR DESCRIPTION
**Pull request by msbemba**
Step 3 can be used for all employees if there is a requirement to have more exceptions w.r.t email like customer rewrite URL etc. as the default policy has exceptions which has  Settings that apply to content except email. , made changes accordingly, So please review and approve.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2501

File: microsoft-365/security/office-365-security/set-up-atp-safe-links-policies.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ After you have reviewed (or edited) the default ATP Safe Links policy that appli
 6. Choose **Save**.
 
 > [!NOTE]
-> ATP Safe Links policies with higher priority will take precedence. If a user is subject to two or more polices, only the higher priority policy will take effect.
+> ATP Safe Links policies with higher priority will take precedence. If a user is subject to two or more polices, only the higher priority policy will take effect. If you want the customer policy to trake precedence, please raise the priority of the policy.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/set-up-atp-safe-links-policies.md]:
```suggestion
> ATP Safe Links policies with higher priority will take precedence. If a user is subject to two or more policies, only the higher priority policy will take effect. If you want the customer policy to take precedence, you need to raise the priority of the policy.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MSFTTracyP Please merge this PR. Thanks.

**Pull request merged**